packages/react-cap-theme/src/theme/CAPTheme.tsx

```diff
@@ -235,6 +175,22 @@ export const CAP_THEME_DEFAULTS = {
   'fixme/ctrl/card/header-padding-outside': tokens.spacingVerticalM,
   'fixme/ctrl/card/header-padding-inside': tokens.spacingVerticalS,
   'fixme/ctrl/card/footer-horizontal-gap': tokens.spacingHorizontalS,
+
```

Comment:
@sxsx1xsxs , @dvdzkwsk , please review the tokens. When a corresponding token was not defined in the design specifications, I used fixme/... as a placeholder.